applications/MeshMovingApplication/python_scripts/mesh_solver_base.py

```diff
@@ -80,17 +86,29 @@ def GetDefaultSettings(cls):
             },
             "reform_dofs_each_step"     : false,
             "compute_reactions"         : false,
+            "poisson_ratio"             : 0.3,
+            "calculate_mesh_velocity"   : false,
```

Comment:
I'd set it to `true`. But I remember that someone complained because it was not needed in some applications of the mesh motion (e.g. optimization).

For me makes much more sense to have the overhead of computing something that it is not needed than having the risk of silently hidind that your advection is not properly computed due to the zero mesh velocity.

Having said this, yes. I'd like this to be `true` by default.

Reply:
I have switched it of for our use cases, so we are independent of this default value.

Reply:
fine for me as well to set it to true, shapeopt is the only application where I know it could cause problems

Reply:
> I have switched it of for our use cases, so we are independent of this default value.

Cool. Let's keep it as `true` by default then.